src/Elastic.Apm.AspNetCore/ApmMiddleware.cs

```diff
@@ -1,112 +1,115 @@
 ﻿using System;
-using System.Threading.Tasks;
-
-using Microsoft.AspNetCore.Http;
 using System.Reflection;
-using Elastic.Apm.Model.Payload;
 using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
 using Elastic.Apm.Helpers;
+using Elastic.Apm.Model.Payload;
+using Microsoft.AspNetCore.Http;
 
-[assembly: InternalsVisibleTo("Elastic.Apm.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010051df3e4d8341d66c6dfbf35b2fda3627d08073156ed98eef81122b94e86ef2e44e7980202d21826e367db9f494c265666ae30869fb4cd1a434d171f6b634aa67fa8ca5b9076d55dc3baa203d3a23b9c1296c9f45d06a45cf89520bef98325958b066d8c626db76dd60d0508af877580accdd0e9f88e46b6421bf09a33de53fe1")]
-[assembly: InternalsVisibleTo("Elastic.Apm.AspNetCore.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010051df3e4d8341d66c6dfbf35b2fda3627d08073156ed98eef81122b94e86ef2e44e7980202d21826e367db9f494c265666ae30869fb4cd1a434d171f6b634aa67fa8ca5b9076d55dc3baa203d3a23b9c1296c9f45d06a45cf89520bef98325958b066d8c626db76dd60d0508af877580accdd0e9f88e46b6421bf09a33de53fe1")]
+[assembly:

```

Comment:
This is unfortunate but is due to our `wrap` of `150` I think.

Reply:
I'd say this is acceptable. Not a big deal... Alternative solution: we could put the `PublicKey` int a `consts`.